content/sensu-go/5.0/guides/troubleshooting.md

```diff
@@ -69,6 +69,28 @@ following those logs are described. The name of the desired service, e.g.
 _NOTE: Platform versions described above are for reference only and do not
 supercede the documented [supported platforms][platforms]._
 
+### Log Messages
+
+#### Permission Issues
+
+Files and folders within `/var/cache/sensu/` need to be owned by the sensu user and group. You will see a logged error similar to the following if there is a permission issue with either the sensu-backend or the sensu-agent:
+
+{{< highlight json >}}
+{"component":"agent","error":"open /var/cache/sensu/sensu-agent/assets.db: permission denied","level":"fatal","msg":"error executing sensu-agent","time":"2019-02-21T22:01:04Z"}
+{{< /highlight >}}
```

Comment:
Here's another error I came across `{"component":"sensu-enterprise","level":"fatal","msg":"error starting etcd: mkdir /var/lib/sensu: permission denied","time":"2019-03-05T20:24:01Z"}`

Reply:
I knew there was another one. I'll get that added as that directory has `sensu-agent` and `sensu-backend` in it, `etcd stuff for the backend. Not sure what the agent uses it for.